packages/myst-cli/src/build/meca/index.ts

```diff
@@ -157,7 +157,7 @@ export async function runMecaExport(
   const toc = tic();
   const { output, articles } = exportOptions;
   // At this point, export options are resolved to contain zero or one articles
```

Comment:
This comment says there may not be articles, and now the next line of code handles that case.